pkgs/by-name/si/simpleBluez/package.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, dbus
+, fmt_9
+, lib
+}:
+let
+  pname = "simpleBluez";
+  version = "0.7.3";
+  src = fetchFromGitHub {
+    owner = "OpenBluetoothToolbox";
+    repo = pname;
```

Comment:
The canonical upstream location for this seems to be <https://github.com/OpenBluetoothToolbox/SimpleBLE/tree/main/simplebluez>, not the archived repository this is pulling from (the builds seem to fail foe this reason...)

Reply:
Ooops, I corrected it but (the hash was the correct one), probably I probably inadvertently reverted the change.